docs/docusaurus/docs/cloud/quickstarts/python_api_quickstart.md

```diff
@@ -0,0 +1,164 @@
+---
+sidebar_label: 'Quickstart for GX Cloud and Python scripts'
+title: 'Quickstart for GX Cloud and Python scripts'
+id: python_quickstart
+description: Connect to a GX Cloud account from a Python script.
+---
+import Prerequisites from '/docs/components/_prerequisites.jsx'
+import PrereqPython from '/docs/components/prerequisites/_python_version.md'
+import SetupAndInstallGx from '/docs/components/setup/link_lists/_setup_and_install_gx.md'
+import DataContextInitializeInstantiateSave from '/docs/components/setup/link_lists/_data_context_initialize_instatiate_save.md'
+
+In this quickstart, you'll learn how to use GX Cloud from a Python script or interpreter, such as a Jupyter Notebook. You'll install Great Expectations, configure your GX Cloud environment variables, connect to sample data, build your first Expectation, validate data, and review the validation results through Python code.
+
+## Data validation workflow
```

Comment:
```suggestion
## Data validation workflow
```
Remove the workflow diagram. It’s not necessary or helpful in a quickstart when the right TOC shows the task sequence. Also, this is inconsistent with other topics and adds visual clutter.

Reply:
Removed in an upcoming commit -- this was a hold over from using the OSS quickstart as a base, which unfortunately is not formatted as tightly as the GX Cloud docs.